packages/rfw/lib/src/flutter/runtime.dart

```diff
@@ -169,6 +171,11 @@ class LocalWidgetLibrary extends WidgetLibrary {
   LocalWidgetBuilder? findConstructor(String name) {
     return _widgets[name];
   }
+
+  /// Returns an unmodifiable map from widget name to LocalWidgetBuilder.
```

Comment:
LocalWidgetBuilder should be a hyperlink.

This should probably include more detail, also. If someone is reading these docs, they have a question. What's the question that might lead them here?

For example, it could be something like

```
/// The widgets defined by this [LocalWidgetLibrary].
///
/// The returned map is an immutable view of the map provided to the constructor.
/// They keys are the unqualified widget names, and the values are the corresponding
/// [LocalWidgetBuilder]s.
///
/// The map never changes during the lifetime of the [LocalWidgetLibrary], but a new
/// instance of an [UnmodifiableMapView] is returned each time this getter is used.
///
/// See also:
///
///  * [createCoreWidgets], a function that creates a [Map] of local widgets.
```

Reply:
Done.